README.md

```diff
@@ -7,26 +8,19 @@ to get started with the Azure Functions extension.
 
 ## Prerequisites
 
-* Install your desired version of the [Azure Functions Core Tools](https://docs.microsoft.com/azure/azure-functions/functions-run-local)
-  * For version 2.0 (This version works cross-platform, but is still in preview and not recommended for production):
-    * Install [.NET Core 2.0](https://www.microsoft.com/net/download/core)
-    * Install the Azure Functions Core Tools:
-    ```bash
-    npm install -g azure-functions-core-tools@core
-    ```
-    > Note: When installing on Ubuntu, you may need to use `sudo`. On MacOS and Linux, you may need to include the `unsafe-perm` flag, as follows:
-    ```bash
-    sudo npm install -g azure-functions-core-tools@core --unsafe-perm true
-    ```
-  * For version 1.0 (This version is only supported on Windows):
-    ```bash
-    npm install -g azure-functions-core-tools
-    ```
-* Install the prerequisites for your desired language:
-  * [JavaScript](#javascript)
-  * [C#](#c)
-  * [Java](#java)
-> NOTE: You may change your `azureFunctions.projectLanguage` user setting to multiple other 'preview' languages not listed above. This allows you to create a project/function in that language, but we do not yet support local debugging for these languages.
+Follow the OS-specific instructions to install the [Azure Functions Core Tools](https://docs.microsoft.com/azure/azure-functions/functions-run-local):
```

Comment:
Do we need to link the doc if all the information is in our README?  Also because the source material seems to be more directly copied from here https://github.com/Azure/azure-functions-core-tools#installing